npm/commander/2.19.0/@n4jsd/commander/index.n4jsd

```diff
@@ -0,0 +1,19 @@
+/**
+ * @see https://github.com/tj/commander.js/
+ */
+export external public class __Command {
+    public version(v: string, customFlag: string=): __Command;
+    public arguments(args: string): __Command
+    public description(desc: string): __Command;
+    public usage(usg: string): __Command
+    public option(opt: string, descr: string, fn: Function | string=): __Command;
```

Comment:
You can also also use a `RegExp`, and additionally define a default, and the defaults may be any value, not just strings (e.g. integers), so I think it's:

`public option(opt: string, descr: string, collector: RegExp|Function| any=, default: any=): Command;`

I know `any` in a union hardly makes sense, but the union gives some notion of how to use the API.